contents/docs/advanced/proxy/netlify.mdx

```diff
@@ -6,23 +6,26 @@ showTitle: true
 
 import RegionWarning from "../_snippets/region-warning.mdx"
 import ProxyWarning from "../_snippets/proxy-usage-warning.mdx"
+import ProxyPathNamesWarning from "../_snippets/proxy-path-names-warning.mdx"
 
 <ProxyWarning />
 
 <RegionWarning />
 
-Netlify supports [redirects and rewrites](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file) which we can use as a reverse proxy from an `/ingest` route. In your `netlify.toml` file, add a redirect like this:
+<ProxyPathNamesWarning />
```

Comment:
Could we just combine these three warnings into one? I know we use some separately, but it might be nice to have like:

> **Notes:
> 1. warning 
> 2. warning
> 3. warning

Rather than have 3 callouts